wework/src/styles/globals.css

```
@@ -156,6 +156,143 @@ button:disabled {
+    margin-inline-start: 2px;
+    color: currentColor;
+    font-size: 10px;
+    font-weight: 650;
+    line-height: 1;
+    pointer-events: none;
+    opacity: 0;
+    animation: file-change-delta-float 560ms ease-out;
+  }
+
+  @keyframes file-change-number-roll {
+    from {
```

Stylelint (17.14.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)